### PR TITLE
Feature/overview columns

### PR DESCRIPTION
**Pull request by Abdalsalaam**
### Description

On the order overview page, I added 2 new columns Delivery Date & Shipping options

### Steps to Test

1. Create an order with delivery date.
2. Create the label for this order with some shipping options.
3. Check the orders list.

image

### CODE REVIEW

File: src/Order/OrdersList.php [hunk 1]
```diff
@@ -49,4 +62,76 @@ public function add_order_barcode_column_content( $column, $order_id ) {
...
+	 * @return void.
+	 */
+	public function add_order_shipping_options_column_content( $column, $order_id ) {
+		if ( $order_id ) {
+			if ( 'postnl_shipping_options' === $column ) {
+				$backend_data = $this->get_backend_data( $order_id );
+
+				echo Utils::generate_shipping_options_html( $backend_data );
+			}
+		}
```
Comment (iyut) [inline src/Order/OrdersList.php:135]:
```suggestion
		if ( empty( $order_id ) ) {
		     return;
		}
	
		if ( 'postnl_shipping_options' === $column ) {
			$backend_data = $this->get_backend_data( $order_id );

			echo Utils::generate_shipping_options_html( $backend_data );
		}
```

Comment (iyut) [reply src/Order/OrdersList.php:135]: Let's try to not use **nested if** as much as possible.

File: src/Order/OrdersList.php [hunk 2]
```diff
@@ -49,4 +62,76 @@ public function add_order_barcode_column_content( $column, $order_id ) {
...
+
+				if ( ! is_a( $order, 'WC_Order' ) ) {
+					return;
+				}
+
+				$delivery_info = $this->get_order_frontend_info( $order, 'delivery_day_date' );
+
+				echo Utils::generate_delivery_date_html( $delivery_info );
+			}
+		}
```
Comment (iyut) [inline src/Order/OrdersList.php:102]:
```suggestion
		if ( empty( $order_id ) ) {
		return;
		}

		if ( 'postnl_delivery_date' === $column ) {
			$order = wc_get_order( $order_id );

			if ( ! is_a( $order, 'WC_Order' ) ) {
				return;
			}

			$delivery_info = $this->get_order_frontend_info( $order, 'delivery_day_date' );

			echo Utils::generate_delivery_date_html( $delivery_info );
		}
```

File: src/Utils.php [hunk 3]
```diff
@@ -532,4 +532,62 @@ function ( $value ) {
...
+	 */
+	public static function get_shipping_options() {
+		return array(
+				'id_check'              => __( 'ID Check', 'postnl-for-woocommerce' ),
+				'insured_shipping'      => __( 'Insured Shipping', 'postnl-for-woocommerce' ),
+				'return_no_answer'      => __( 'Return if no answer', 'postnl-for-woocommerce' ),
+				'signature_on_delivery' => __( 'Signature on Delivery', 'postnl-for-woocommerce' ),
+				'only_home_address'     => __( 'Only Home Address', 'postnl-for-woocommerce' ),
+				'letterbox'             => __( 'Letterbox', 'postnl-for-woocommerce' ),
+				'packets'               => __( 'Packets', 'postnl-for-woocommerce' ),
```
Comment (iyut) [inline src/Utils.php:590]:
```suggestion
				'id_check'              => esc_html__( 'ID Check', 'postnl-for-woocommerce' ),
				'insured_shipping'      => esc_html__( 'Insured Shipping', 'postnl-for-woocommerce' ),
				'return_no_answer'      => esc_html__( 'Return if no answer', 'postnl-for-woocommerce' ),
				'signature_on_delivery' => esc_html__( 'Signature on Delivery', 'postnl-for-woocommerce' ),
				'only_home_address'     => esc_html__( 'Only Home Address', 'postnl-for-woocommerce' ),
				'letterbox'             => esc_html__( 'Letterbox', 'postnl-for-woocommerce' ),
				'packets'               => esc_html__( 'Packets', 'postnl-for-woocommerce' ),
```

Let's make it more secure

**Review by iyut — CHANGES_REQUESTED** [review]
Overall the code looks good. I just left some comment about small code cosmetic

**Comment by Abdalsalaam** [comment]
> Overall the code looks good. I just left some comment about small code cosmetic

Thank you @iyut , I made some changes

**Review by iyut — APPROVED** [review]
Looks good!

**Pull request merged**